Formula/claws-mail.rb

```diff
@@ -0,0 +1,26 @@
+class ClawsMail < Formula
+  desc "The user-friendly, lightweight, and fast email client"
+  homepage "https://www.claws-mail.org/"
+  url "https://www.claws-mail.org/releases/claws-mail-3.17.5.tar.gz"
```

Comment:
What is the license for this?

Reply:
It looks like GPLv3

Reply:
Yes, it is GPLv3 (https://www.claws-mail.org/COPYING).